backend/portal/api/portal_api.py

```diff
@@ -259,7 +260,15 @@ def _collection_to_response(collection: CollectionVersionWithDatasets, access_ty
         # The last case is a published collection. We just return the canonical id and set revision_of to None
         revision_of = None
         collection_id = collection.collection_id.id
-        is_in_revision = False
+        is_published = collection.published_at is not None
+
+        _revising_in = None
+        if is_published and access_type == "WRITE":  # can ony be WRITE if authenticated
+            _revising_in = get_business_logic().get_unpublished_collection_version_from_canonical(
+                collection.collection_id
+            )
+        revising_in = _revising_in.version_id.id if _revising_in else None
+        is_in_revision = True if _revising_in else None
```

Comment:
nit: change `is_in_revision` to `has_outstanding_revision` or something more explicit like that